planning/behavior_path_planner/README.md

```diff
@@ -253,39 +253,118 @@ The Pull Out module is activated when ego-vehicle is stationary and footprint of
-  - Generated Path is safe (This condition is temporarily invalid)
-    - It is possible to enable or disable this condition. (default: `disabled`)
+  - It is always ready if the conditions of the request are met.
+
+- Pull out start condition
+  - Generate safe path which does not collide with obstacles
     - If safe path cannot be generated from the current position, safe path from a receding position is searched.
   - Pull out is allowed by an operator
 
 #### **finish pull out condition** (need to meet any of the conditions below)
 
-- The distance to the goal of your vehicle is lower than threshold (default: < `1m`)
-- The speed of the vehicle is 0.
+- Exceeding the pull out end point by more than the threshold (default: `1.0m`)
+
+#### General parameters for pull_over
+
+| Name                         | Unit  | Type   | Description                                                    | Default value |
+| :--------------------------- | :---- | :----- | :------------------------------------------------------------- | :------------ |
+| th_arrived_distance_m        | [m]   | double | distance threshold for arrival of path termination             | 1.0           |
+| th_stopped_velocity_mps      | [m/s] | double | velocity threshold for arrival of path termination             | 0.01          |
+| th_stopped_time_sec          | [s]   | double | time threshold for arrival of path termination                 | 1.0           |
+| collision_check_margin       | [m]   | double | Obstacle collision check margin                                | 1.0           |
+| pull_out_finish_judge_buffer | [m]   | double | threshold for finish judgment distance from pull out end point | 1.0           |
 
 #### **Safe check with obstacles in shoulder lane**
 
-1. Calculate ego-vehicle's footprint on pull out path between from current position to merge end point. (Illustrated by blue frame)
+1. Calculate ego-vehicle's footprint on pull out path between from current position to pull out end point. (Illustrated by blue frame)
 2. Calculate object's polygon which is located in shoulder lane
-3. If a distance between the footprint and the polygon is lower than the threshold (default: `1.5 m`), that is judged as a unsafe path
+3. If a distance between the footprint and the polygon is lower than the threshold (default: `1.0 m`), that is judged as a unsafe path
 
-![pull_out](./image/pull_out_fig1.drawio.svg)
+![pull_out_collision_check](./image/pull_out_collision_check.drawio.svg)
 
 #### **Path Generation**
 
-The path is generated with a certain margin from the left edge of the shoulder lane.
+There are two path generation methods.
+
+##### **shift pull out**
+
+Pull out distance is calculated by the speed, lateral deviation, and the lateral jerk. The lateral jerk is searched for among the predetermined minimum and maximum values, and the one satisfies ready conditions described above is output.
```

Comment:
```suggestion
Pull out distance is calculated by the speed, lateral deviation, and the lateral jerk. The lateral jerk is searched for among the predetermined minimum and maximum values, and the one that satisfies the ready conditions described above is selected.
```

It is said, "It is always ready if the conditions of the request are met" above. What do you mean "satisfies ready conditions described above"?

Reply:
This is wrong, I just brought up old docs sentences. Fixed in https://github.com/autowarefoundation/autoware.universe/pull/1587/commits/3fe45f3a0539707e19b52304e43c03d3a05e6092.

Reply:
"It is always ready if the conditions of the request are met" is true.